artifacts/fedora/fedora.go

```diff
@@ -119,6 +124,7 @@ func (f *fedoraGatherer) Gather() ([][]api.Artifact, error) {
 				[]api.Artifact{
```

Comment:
That does not look right to me? It will add a slice with one `Artifact` for each combination of `release.Version` and `release.Arch` to `artifacts`?

Reply:
This loop should iterate over each found release and in an inner loop over every architecture. This way you can build a slice of `Artifacts` for a single version with different architectures and avoid hardcoding versions into the Gatherer.